### ISSUE TITLE
Purchase report in store is wrong

### ISSUE THREAD

**Issue by angband-import** [angband/angband#773]
**Reported by ctate on 9 Jul 2007 05:47 UTC**
When purchasing a stackable item from a store, it's supposed to report the number of items purchased in the transaction, followed by the number of that item now in your inventory.  In 3.0.8 there's a bug in the second message:  it's repeating the number purchased, not reporting the number you now hold.  For example, this just happened to me in the General Store:

```
Agreed!
You have bought 2 Rations of Food (c) for 8 gold.
You have 2 Rations of Food (b).
Agreed!
You have bought 2 Rations of Food (c) for 8 gold.
You have 2 Rations of Food (b).
```

[Checking my inventory at this point shows 4 rations of food, as it should be, but the message was wrong.]

**Comment by angband-import** [comment]
**Modified by ajps on 13 Jul 2007 17:36 UTC**

**Comment by angband-import** [comment]
**Comment by ajps on 13 Jul 2007 18:07 UTC**
Fixed in [335] merged in [5dfe994] (SVN r399).
